.github/workflows/error-ref-publisher.yaml

```diff
@@ -0,0 +1,63 @@
+name: Meshery-Istio Error Codes Utility Runner
```

Comment:
changed the name to Meshery-Istio, just wondering if this is correct(?) @DelusionalOptimist

Reply:
Umm yeah we named the utility "meshkit error codes utility" and it would be easier to find the runs of this workflow if we name it generically across all repos. So I guess this should stay like how it was earlier :stuck_out_tongue:

Reply:
I see, lemme change it real quick